scripts/update-index-alias.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python
+"""
+A script to update the search index an alias is assoiciated with.
+
+To delete old indexes (those with no alias), set --delete-old-indexes=yes
+
+Usage:
+scripts/update-index-alias.py <alias> <target> <stage> <search-api-endpoint> [options]
+
+Options:
+    --delete-old-indexes=<delete-old-indexes>  [default: no]
+"""
+import os
+import sys
+import subprocess
+import yaml
+import json
+import requests
+from requests.exceptions import HTTPError
+from docopt import docopt
+
+
+def update_index_alias(alias, target, stage, endpoint, delete_old_indexes):
+    auth_token = _get_auth_token(stage)
+    headers = {
+        'Authorization': "Bearer {}".format(auth_token),
+        'Content-type': 'application/json'
+    }
+
+    _rename_current_alias_to_old(alias, endpoint, headers)
+
+    url = "https://{}/{}".format(endpoint, alias)
+    data = {'type': 'alias', 'target': "{}".format(target)}
+
+    response = requests.put(url, headers=headers, json=data)
+    _check_response_status(response, 'updating alias')
+
+    if delete_old_indexes == 'yes':
```

Comment:
I think it would be nice to be a bit more forgiving here. Maybe use `if strtobool(delete_old_indexes)` using `from distutils.util import strtobool`. Alternatively use argparse with that function as the type (my preferred), but either/or would be good.